veracruz-mcu-client/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM ubuntu:18.04
+
+ARG TARGET
+ARG USER
+ARG UID
+
+ENV DEBIAN_FRONTEND noninteractive
+# Needed for Zephyr's tools
+ENV PYTHONIOENCODING "UTF-8"
+
+# Need different version of cmake from kitware
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        gnupg \
+        software-properties-common \
+        wget && \
+    wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | apt-key add - && \
+    apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main' && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+        cmake=3.18.4-0kitware1 \
+        cmake-data=3.18.4-0kitware1
+
+# Zephyr dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ccache \
+        device-tree-compiler \
+        dfu-util \
+        file \
+        g++-multilib \
+        gcc \
+        gcc-multilib \
+        git \
+        gperf \
+        libsdl2-dev \
+        make \
+        ninja-build \
+        protobuf-compiler \
+        python3-dev \
+        python3-pip \
+        python3-protobuf \
+        python3-setuptools \
+        python3-tk \
+        python3-wheel \
+        wget \
+        xz-utils
+
+# Install Zephyr
+WORKDIR /zephyr-workspace
+RUN pip3 install west && \
+    west init --mr v2.4.0 && \
+    west update && \
+    pip3 install -r zephyr/scripts/requirements.txt
+
+# Get the Zephyr SDK
+#
+# TODO we don't need all the compilers in this package... can we reduce
```

Comment:
Is this TODO still relevant?  If so, can you open up an issue in the issue tracker to make sure it isn't forgotten?

Reply:
I think a single cover-all issue is OK, yeah.

Reply:
I think I'm going to actually open up one for each. There's not _that_ many, they can then be closed individually, and labels should help with grouping the related issues together